bootstrap.sh

```diff
@@ -239,6 +239,13 @@ EOF
 
   echo "You may want to install the following:"
   sudo softwareupdate -l -a
+
+  git clone https://github.com/travis-ci/system-info.git
+  cd system-info
+  bundle install
+  bundle exec ruby exe/system-info
```

Comment:
do you think that we need to have `--cookbooks-sha` defined in this case, since we're not even using chef to provision these images? (yet)